File: simple/java/replace-nested-conditional-with-guard-clauses_before.java
public double getPayAmount() {
  if (_isDead)
    result = deadAmount();
  else {
    if (_isSeparated)
      result = separatedAmount();
    else {
      if (_isRetired)
        result = retiredAmount();
      else
        result = normalPayAmount();
    }
  }
  return result;
}

File: simple/csharp/replace-nested-conditional-with-guard-clauses_before.cs
public double GetPayAmount()
{
  double result;
  
  if (isDead)
  {
    result = DeadAmount();
  }
  else 
  {
    if (isSeparated)
    {
      result = SeparatedAmount();
    }
    else 
    {
      if (isRetired)
      {
        result = RetiredAmount();
      }
      else
      {
        result = NormalPayAmount();
      }
    }
  }
  
  return result;
}